### ISSUE TITLE
Syntax error - Can't compile?

### ISSUE THREAD

**Issue by EKennedyAustralia** [twilio-labs/plugin-queued-callbacks-and-voicemail#4]
InQueueVoicemailComponent.js: JSX value should be either an expression or a quoted JSX text (300:12)

![image](https://user-images.githubusercontent.com/55265321/98327526-d5670f00-2047-11eb-903d-c751fee41359.png)

That's if I do twilio flex:plugins:start to run locally.
Twilio flex:plugins:deploy just returns failed to compile with no other output

**Comment by functionalStoic** [comment]
Hi @EKennedyAustralia,
It appears the value of the prop `InputProps` right above has been cleared. It should look like this:

```
...
InputProps={{
  style: {
    fontSize: '9pt',
  },
}}
value={transcriptText}
...
```
Here it is in the full form:
https://github.com/twilio-labs/plugin-queued-callbacks-and-voicemail/blob/39847dd2d429a131886a6f63c5136c1e259fb3d2/src/components/InQueueMessaging/InQueueVoicemailComponent.js#L299

Let me know if that get's you unblocked

**Comment by jsandersj** [comment]
Hi @functionalStoic ,

I am not original poster, but I had same issue, and your fix above resolved my issue. Thank you!

**Comment by functionalStoic** [comment]
Hi @jsandersj, I appreciate you following up with that confirmation! I hope it resolved the issue for @EKennedyAustralia as well 🤞 
Any idea how you ran into that issue? I'm curious about the steps that lead you there.
Thanks!

**Comment by jsandersj** [comment]
Unknown. I followed the instructions here:

https://www.twilio.com/docs/flex/solutions-library/queued-callback-and-voicemail#download-the-flex-queued-callbacks-and-voicemail-plugin

I downloaded the plugin using this syntax from that URL:

twilio flex:plugins:create plugin-queued-callbacks-and-voicemail --template https://github.com/twilio-labs/plugin-queued-callbacks-and-voicemail/tree/main

I hit this error on Windows, so I tried on Linux, same issue. It is not a straight git clone like in the readme, maybe something in how the twilio CLI does its thing? I wasn't even sure that separate js file existed until I saw your note here, definitely did not make any manual changes there. Sorry I can't be of more help.
